config/routes.rb

```diff
@@ -11,4 +11,15 @@
   resources :account_activations, only: %i(edit)
   resources :password_resets, only: %i(new create edit update)
   resources :microposts, only: %i(create destroy)
+  resources :relationships, only: %i(create destroy)
+  resources :followings do
```

Comment:
```suggestion
  resources :followings, only: :index
```
thế này thôi là được